Python/Product/PythonTools/PythonTools/Environments/AddCondaEnvironmentControl.xaml

```diff
@@ -181,7 +182,10 @@
                         x:Name="ProjectComboBox"
                         Margin="0 0 0 17"
                         MinHeight="29"
-                        Width="249"
+                        Width="{Binding 
```

Comment:
Some of these comboboxes were ending up too wide in the maximum scaling situation (Screen is only 600px wide in that scenario)

Reply:
i can't remember if there is an auto width based on percentage of parent

Reply:
Not that I could find. All the examples I found referenced writing a converter

Reply:
There is a possibility of using a grid instead. That might work too. Combobox would have to be set to full stretch (and that can be difficult given it doesn't do that by default - you have to override the template)

For example this is one of the comboboxes set to stretch (it only stretches to fill the content)

![image](https://user-images.githubusercontent.com/19672699/129978835-8f702da4-372b-47e0-b08e-05049238f7a3.png)

Which must be why Hugue forced the width.

Reply:
The solution here is probably simpler than generating a new template for the different controls.

Reply:
here is the example i found. but will go with what you have
```
<Grid >
    <Grid.ColumnDefinitions>
        <ColumnDefinition Width="2*" /><!--This will make any control in this column of grid take 2/5 of total width-->
        <ColumnDefinition Width="3*" /><!--This will make any control in this column of grid take 3/5 of total width-->
    </Grid.ColumnDefinitions>
    <Grid.RowDefinitions>
        <RowDefinition MinHeight="30" />
    </Grid.RowDefinitions>

    <TextBlock Grid.Column="0" Grid.Row="0">Your text block a:</TextBlock>
    <TextBlock Grid.Column="1" Grid.Row="0">Your text block b:</TextBlock>
</Grid>
```